mindmeld/models/model.py

```diff
@@ -44,6 +44,7 @@
     get_label_encoder,
     ingest_dynamic_gazetteer,
     register_label,
+    SENTIMENT_ANALYZER,
```

Comment:
Nit: maybe regroup this var closer to the other uppercase vars

Reply:
Done